backend/code-style.md

```
@@ -0,0 +1,175 @@
+<li>success</li>
+<li>error</li>
+</ul>
+
+ 항목           | 규칙            
+--------------|---------------
+ @Nested      | ⭕             
+ @DisplayName | 시나리오·조건·결과 명시 
+
+## **2.2. BDD 패턴**
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)